[opentelemetry][ci-cd] add section for Ansible tower (#1722)
elastic/observability-docs: docs/en/observability/ci-cd-observability.asciidoc

.ServiceMap view of a Jenkins pipeline execution instrumented with the Ansible plugin
image::images/ansible-service-map.png[Ansible service map view]

> 

Reviewer: This should fix the failing doc build

```suggestion
[discrete]
```

[[ci-cd-ansible-tower]]
=== Ansible AWX/Tower

> The Ansible OpenTelemetry plugin integration supports Ansible AWX/Tower, if you'd like to enable it please follow the below steps.

Reviewer: ```suggestion
The Ansible OpenTelemetry plugin integration supports Ansible AWX/Tower. To enable it, follow these steps.
```

AWX requires an Execution Environment with the Ansible and Python packages installed.

> For this we used the https://www.ansible.com/blog/introduction-to-ansible-builder tool to create the container definition.

Reviewer: ```suggestion
You can use the https://www.ansible.com/blog/introduction-to-ansible-builder[Absible Builder CLI tool] to create the container definition.
```

> 
> You then upload the container to an image repository accessible by AWX and define an Execution Environment using the container you created.

Reviewer: ```suggestion
Then upload the container to an image repository accessible by AWX and define an Execution Environment using the container you created.
```

> To inject the environment variables and service details you can use custom credential types and then assign the credentials to the Playbook template.
> This gives you the flexibility to reuse the endpoint details for Elastic APM and also standardize on custom fields for reporting purposes.

Reviewer: ```suggestion
To inject the environment variables and service details, use custom credential types and assign the credentials to the Playbook template.
This gives you the flexibility to reuse the endpoint details for Elastic APM and also standardize on custom fields for reporting purposes.
```

[role="screenshot"]
.An APM Service Endpoint in Ansible AWX/Tower
image::images/ansible-automation-apm-endpoint.png[APM Service Endpoint in Ansible Tower]

[role="screenshot"]
.Custom fileds in Ansible AWX/Tower
image::images/ansible-automation-apm-service-details.png[Custom fields in Ansible Tower]

> Finally, https://www.elastic.co/blog/5-questions-about-ansible-that-elastic-observability-can-answer provides a great
> overview of how all those pieces work together.

Reviewer: ```suggestion
Want to learn more? This https://www.elastic.co/blog/5-questions-about-ansible-that-elastic-observability-can-answer[blog post] provides a great
overview of how all of these pieces work together.
```
